models/bear.js

```diff
@@ -0,0 +1,31 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const Joi = require('joi');
+
+//Schema for the bear
+const bearSchema = new mongoose.Schema({
+  name: String,
+  age: {
+    type: Number,
+    min: 1,
+  },
+  color: String,
+  species: String
+})
+
+//Bear collection model
+const Bear = mongoose.model('bears', bearSchema);
+
+//validates the input
+function validateBear(bear){
+  const schema = {
+    name: Joi.string(),
+    age: Joi.number(),
+    color: Joi.string(),
+    species: Joi.string()
+  }
+  return Joi.validate(bear, schema);
+}
+
+exports.Bear = Bear;
+exports.validateBear = validateBear;
```

Comment:
It may be a lot better if you export a single object containing all the entities you want to export and use destructuring to access them in the files you need them

Reply:
ok Boss.